about.html

```diff
@@ -10,36 +10,39 @@
     <div id="about-profile" class="text-center">
         <img id="profile-pic" src="/assets/logo-large.png" alt="CSS Logo">
         <h1>Computer Science Society</h1>
-        <a href="https://cssbham.com/discord" class="button" id="discord-button"><i class="fab fa-discord"></i> Join Discord</a>
-        <a href="https://cssbham.com/join" class="button" id="guild-button"><img src="/assets/about/guild-logo.svg" class="text-img" alt=""> Get Membership</a>
+        <a href="https://cssbham.com/discord" class="button" id="discord-button"><i class="fab fa-discord"></i> Join
+            Discord</a>
```

Comment:
These newlines are unnecessary

Reply:
Oops, that's IntelliJ auto formatting, i can remove those if you want